README.md

```diff
@@ -0,0 +1,85 @@
+# msu-frontend
+
+![MSU](http://msucongress.com/wp-content/uploads/2017/10/msulogominutes.jpg)
+
+Hey there!
+
+This is the official code that is used to create the frontend (the screen layout) of the Marianopolis Student Union guide.
+
+To encourage tech literacy for people of all backgrounds, this is the guide that uses simple terms to describe how to download and run this app.
+
+## Installing the Right Tools
+
+This project will NOT work if you do not install Git, NodeJS/npm or React Native.
+
+Scary? Not at all, just follow the instructions on the websites below to install the two tools.
+
+### Git (https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+Used for version control, simply just keeping track of progress
+
+### NodeJS and npm (https://www.npmjs.com/get-npm)
+JavaScript library and installation tool
+
+Next, install React-Native with npm by typing `npm install -g react-native-cli` in cmd (Windows) or Terminal (MacOS or Linux). This command installs React Native globally on your computer for all your projects.
+
+## Downloading the Frontend
+
+Now, this is the cool part. Open cmd (Windows) or Terminal (MacOs or Linux).
+
+Go to Desktop (or any given directory you want), which you can navigate by using:
+
+* `dir` or `ls` which displays all the files in your current folder
+* `pwd` to print the path of your current folder
+* `cd <folder name>` to go to a new folder whose name is in `<folder name>`
+* `cd ..` to go back one folder
+
```

Comment:
Don't need to provide a basic shell tutorial in every repository that requires the tiniest bit of terminal usage. Anyone doing this sort of stuff will already know this.